src/test/java/org/elasticsearch/test/InternalTestCluster.java

```diff
@@ -181,8 +181,8 @@
 
     private static final boolean ENABLE_MOCK_MODULES = RandomizedTest.systemPropertyAsBoolean(TESTS_ENABLE_MOCK_MODULES, true);
 
-    static final int DEFAULT_MIN_NUM_DATA_NODES = 2;
-    static final int DEFAULT_MAX_NUM_DATA_NODES = 6;
+    static final int DEFAULT_MIN_NUM_DATA_NODES = 1;
+    static final int DEFAULT_MAX_NUM_DATA_NODES = TEST_NIGHTLY ? 6 : 3;
```

Comment:
I decided to go with 3 since I think unless we have nightly runs enabled which we are not at this point we loosing a lot of coverage